src/main/java/com/commercetools/sync/products/utils/ProductUpdateActionUtils.java

```diff
@@ -838,6 +848,45 @@ public static List<UpdateAction<Product>> buildChangeMasterVariantUpdateAction(
         });
   }
 
+  private static List<UpdateAction<Product>> buildSetAttributeInAllVariantsAction(
```

Comment:
A comment would be nice here.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/918/files#diff-c23a4d1680c961a86cbb2611de8a67213974d2b8bfd49f999013e04e39de40f7R497-R499